elegant/process_experiment.py

```diff
@@ -80,11 +81,23 @@ def segment_experiment(experiment_root, model, channels='bf', use_gpu=True, over
         overwrite_existing: if False, the segmenter will not be run on existing
             mask files, nor will existing annotations be modified even if new
             mask files are generated for a timepoint.
+        timepoint_filter - filter for scan_experiment_dir specifying timepoints to segment
+        mask_root - root directory for masks that will be used to updating annotations
```

Comment:
What's the use-case for specifying a `mask_root` outside of the experiment directory? If none, I recommend this be `mask_dir` instead, with a default of `'derived_data/mask'`, which then gets attached to `experiment_root`.

Reply:
Resolved